prediction_market_agent_tooling/markets/seer/seer.py

```
@@ -281,14 +295,57 @@
+        api_keys: APIKeys | None = None,
+        web3: Web3 | None = None,
+    ) -> str:
+        """
+        Sells the given number of shares for the given outcome in the given market.
+        """
+        outcome_token = self.get_wrapped_token_for_outcome(outcome)
+        api_keys = api_keys if api_keys is not None else APIKeys()
+        amount_to_trade = xdai_type(amount.amount)
+
+        order_metadata = swap_tokens_waiting(
+            amount_wei=xdai_to_wei(amount_to_trade),
```

Ruff (0.8.2)

Undefined name `xdai_to_wei`

(F821)